src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -775,6 +778,43 @@ namespace ngraph
                         quantized_conv_bias->set_op_annotations(op_annotations);
                     }
                 }
+
+                template <>
+                void CPUAssignment::ASSIGN_DECL(ngraph::op::Dequantize)
+                {
+                    auto dequantize = static_cast<op::Dequantize*>(node);
+                    auto offset_const_op =
+                        std::static_pointer_cast<ngraph::op::Constant>(dequantize->get_argument(2));
+                    float offset = *(static_cast<float const*>(offset_const_op->get_data_ptr()));
+                    if ((node->get_input_element_type(0) == element::u8 ||
+                         node->get_input_element_type(0) == element::i8) &&
+                        offset == 0)
+                    {
+                        auto dequantize = static_cast<op::Dequantize*>(node);
+                        auto op_annotations =
+                            std::make_shared<ngraph::runtime::cpu::CPUOpAnnotations>();
+                        op_annotations->set_mkldnn_op(true);
+                        dequantize->set_op_annotations(op_annotations);
+                    }
+                }
+
+                template <>
+                void CPUAssignment::ASSIGN_DECL(ngraph::op::Quantize)
+                {
+                    auto quantize = static_cast<op::Quantize*>(node);
+                    auto offset_const_op =
+                        std::static_pointer_cast<ngraph::op::Constant>(quantize->get_argument(2));
+                    float offset = *(static_cast<float const*>(offset_const_op->get_data_ptr()));
+                    op::Quantize::RoundMode round_mode = quantize->get_round_mode();
+                    if (node->get_input_element_type(0) == element::f32 && offset == 0 &&
+                        round_mode == op::Quantize::RoundMode::HALF_TO_EVEN)
```

Comment:
seems like there should be a "TODO" here at least.  need to hook up MKLDNN directly to Quantize round mode once HALF_TO_EVEN is supported.  need to fall back to reference kernel for HALF_AWAY_FROM_ZERO.